sagemaker-python-sdk/keras_script_mode_pipe_mode_horovod/tensorflow_keras_CIFAR10.ipynb

```diff
@@ -0,0 +1,618 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Make some predictions\n",
+    "To verify the endpoint's functionality, we'll start by simply generate random data in the correct shape and make a prediction."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Creating fake prediction data\n",
+    "import numpy as np\n",
+    "data = np.random.randn(1, 32, 32, 3)\n",
+    "print(\"Predicted class is {}\".format(np.argmax(predictor.predict(data))))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Calculating accuracy and create a confusing matrix based on the test dataset\n",
+    "\n",
+    "Our endpoint works as expected, we'll now use the test dataset for predictions and calculate our model accuracy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from keras.datasets import cifar10\n",
+    "from keras.preprocessing.image import ImageDataGenerator\n",
+    "from sklearn.metrics import confusion_matrix\n",
+    "datagen = ImageDataGenerator()\n",
+    "\n",
+    "(x_train, y_train), (x_test, y_test) = cifar10.load_data()\n",
+    "\n",
+    "def predict(data):\n",
+    "    predictions = predictor.predict(data)\n",
+    "    return predictions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "batch_size = 256\n",
+    "predicted = []\n",
+    "actual = []\n",
+    "batches = 0\n",
+    "for data in datagen.flow(x_test,y_test,batch_size=batch_size):\n",
+    "    for i,prediction in enumerate(predict(data[0])):\n",
+    "        predicted.append(np.argmax(prediction))\n",
+    "        actual.append(data[1][i][0])\n",
+    "    batches += 1\n",
+    "    if batches >= len(x_test) / batch_size:\n",
+    "        break"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cm = confusion_matrix(y_pred=predicted,y_true=actual)\n",
+    "TP = cm[0][0]\n",
+    "FP = cm[0][1]\n",
+    "TN = cm[1][1]\n",
+    "FN = cm[1][0]\n",
+    "accuracy = (TP + TN)/(TP+FP+TN+FN)\n",
+    "display('TP : {}'.format(TP))\n",
+    "display('FP : {}'.format(FP))\n",
+    "display('TN : {}'.format(TN))\n",
+    "display('FN : {}'.format(FN))\n",
+    "display('Accuracy: {}%'.format(round(accuracy*100,2)))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import seaborn as sn\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "n_cm = cm / cm.astype(np.float).sum(axis=1)\n",
+    "sn.set(rc={'figure.figsize':(11.7,8.27)})\n",
+    "sn.set(font_scale=1.4)#for label size\n",
+    "sn.heatmap(n_cm, annot=True,annot_kws={\"size\": 10})# font size"
```

Comment:
Heatmap is not plotting.

Reply:
Updated with 
%matplotlib inline
to allow inline plotting